lang/cs/Org.Apache.REEF.Network/Naming/Events/NamingEvent.cs

```diff
@@ -25,7 +25,7 @@ namespace Org.Apache.REEF.Network.Naming.Events
     /// Event representing a lookup, registering, or unregistering of 
     /// an identifier with the Name Service.
     /// </summary>
-    internal class NamingEvent
+    internal abstract class NamingEvent
```

Comment:
Sorry, I meant that it is subclassed, and instances of its subclasses are created, but no instance of itself is created, which is why I opted to make it an abstract class.